.github/workflows/gqcp.yml

```diff
@@ -0,0 +1,114 @@
+name: GQCP
+
+on: [pull_request]
+
+# These variables are for the slack notification, the default is failure, if all build steps are successful they will be updated to green and "Build successful"
+env:
+  BUILD_COLOR: '#FF0000'
+  BUILD_MESSAGE: 'Build failed'
+
+# Jobs are ran in parallel
+jobs:
+
+# BUILD
+  build:
+    runs-on: ubuntu-latest
+    # Environment variables
+    env:
+      CONDA: /usr/share/miniconda
+      LIBINT_DATA_PATH: $CONDA/share/libint/2.3.1/basis
+
+# Various steps to perform the build
+    steps:
+    # Cmake & Python scripts
+    - uses: snickerbockers/cmake-configure@prerel1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.7' 
+    - uses: actions/checkout@v1
+
+# Set cache actions: links files at a path to a key to be stored
+#    - name: Cache Conda
+#      id: cache-conda  # cache id
+#      uses: actions/cache@v1.0.3. # Marketplace script
+      # Store all files in $CONDA, link with key "Linux-conda"
+#      with:
+#        path: $CONDA
+#        key: ${{ runner.os }}-conda
+
+    - name: Install Miniconda
+ #     if: steps.cache-conda.outputs.cache-hit != 'true'  # If cached skip
+      run: |
+        rm -rf $CONDA;
+        mkdir -p $HOME/download;
+        wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O $HOME/download/miniconda.sh;
+        bash $HOME/download/miniconda.sh -b -p $CONDA;
+
+    - name: Configure Conda
```

Comment:
I would merge this into the previous step: "Install and configure Conda"

Reply:
This is separated for caching